src/commands/mod.rs

```diff
@@ -485,6 +485,10 @@ struct DuplicateArgs {
 /// Abandon a revision, rebasing descendants onto its parent(s). The behavior is
 /// similar to `jj restore --changes-in`; the difference is that `jj abandon`
 /// gives you a new change, while `jj restore` updates the existing change.
+///
+/// Note that if you abandon the working-copy commit, a new empty commit will be created
+/// as a new working-copy commit.
```

Comment:
nit: can be simplified a bit as "a new empty working-copy commit will be created"

Reply:
thank you, simpler is always better!